README.rst

```diff
@@ -133,6 +133,29 @@ input queries before sending them to the API:
 Run ``python -m autoextract --help`` to get description of all supported
 options.
 
+Errors
+~~~~~~
+
+The following errors could happen while making requests:
+
+- Network errors
+- `Request-level errors`_
+    - Authentication failure
+    - Malformed request
+    - Too many queries in request
+    - Request payload size is too large
+- `Query-level errors`_
+    - Downloader errors
+    - Proxy errors
+    - ...
+
+Some errors could be retried while others don't. For example, you could retry
```

Comment:
Alternative:
`
Some errors could be retried because there are chances the response will change in further requests, but for others, it makes no sense. 
`

Reply:
I've updated this part os the readme file.